### DISCUSSION TITLE
Using mods and configs with cubecoders AMP

### DISCUSSION THREAD

**Discussion by cummingsk05** [kus/cs2-modded-server#37] [General]
Not sure if this is the right place to post this, but I'm wondering if there is a way to integrate this with CubeCoders AMP solution. I like the AMP webUI for server management and admin, but would also love to implement the mods, maps, and other configurations available in this repo.

**Comment by immattRZ** [comment]
Looking for this too

**Comment by ghost** [comment]
yeah, that would be cool *bump*

**Comment by bkwar201** [comment]
Bump for another request of this. Tried to put this on AMP and went downhill real fast >_< probably spent a couple hours trying to figure out why it was not working.
